lib/rubocop/cop/rspec/described_class.rb

```diff
@@ -143,8 +145,7 @@ def full_const_name(node)
         #   collapse_namespace([:A, :B], [nil, :C)     # => [nil, :C]
         #   collapse_namespace([:A, :B], [nil, :B, :C) # => [nil, :B, :C]
         def collapse_namespace(namespace, const)
-          return const if namespace.empty?
-          return const if const.first.nil?
+          return const if namespace.empty? || const.first.nil?
```

Comment:
I suspect this change is made _only_ to “please the cop”. I would rather increase the max file length than make unnecessary changes.